tests/js/common/shell/shell-index-geo.js

```diff
@@ -306,7 +307,7 @@ function GeoIndexErrorHandlingSuite() {
       collection.replace(d3, { loc : [ 0 ] });
       collection.replace(d4, { loc : [ -100, -200 ] });
 
-      assertEqual(1, collection.near(0,0).toArray().length);
+      assertEqual(2, collection.near(0,0).toArray().length);
```

Comment:
Why was the test adjusted?

Reply:
Because now we have two valid points

Reply:
Understood. It would have been helpful for me to get the information what actual change made the point valid. I have now spent approx. one hour to look for it. In the end, this leads to massively prolonged times for reviews and is a bit frustrating. Anyway, is resolved now.

Reply:
Everywhere where lat lon point constructed from another points, or from user input, I add assert or normalized.

I think here it's from parseImpl(..., S2LatLng& point) from Geo/Json.cpp

Reply:
Or maybe from old line 127 in Index.cpp